whitelabelco/sofo-foods
# cp/feat/remove-team-section

### Description 💻

Resolves: #155 

### Implementation 💡

Removes Team section from About Us page

I only commented out the code in case the client decides they want the section back in the next version

#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "About Us"
- confirm the Team Section no longer displays

### Screenshots/Demos 📷

### Checklist ✅
